Spellcheck translatable strings

afmenez opened the pull request:
# Description
Some words had inconsistent spelling or casing.  Others where simply wrong.

yw4z commented:
Thanks for working on this side. FYI you can also check these

• All file extension descriptions on UI should use uppercase like STL 3MF etc
• Menu items should have " ..." after its name if they open an external dialog like open a file
replace with STL opens a dialog but not uses " ...". and "Export as one STL" doesnt have space before 3 dots
<img width="281" height="118" alt="Screenshot-20251102182030" src="https://github.com/user-attachments/assets/e1f6ad29-a52c-47cf-a173-4fa6233ca9f5" />
correct examples
<img width="428" height="50" alt="Screenshot-20251102182217" src="https://github.com/user-attachments/assets/7f424c38-c3ef-461a-aa94-ec1de1a7d7b5" />
Also these doesnt have spacing. you can see "L" and "..." starts to mixing. "STL ..." vs "STL..."
<img width="527" height="211" alt="Screenshot-20251102182433" src="https://github.com/user-attachments/assets/eddc77e8-3147-43a5-a8ec-72d338b5b2d5" />
<img width="576" height="92" alt="Screenshot-20251102182438" src="https://github.com/user-attachments/assets/279ecd28-1d61-4909-a0be-3b4f1a71e01e" />

• You can check "Sentence case" usage. i'm sure there are many more. only first letter should use uppercase except some special names
<img width="376" height="65" alt="Screenshot-20251102182811" src="https://github.com/user-attachments/assets/6cf61b72-0f1a-4d40-9908-e8325615d8f1" />
<img width="399" height="72" alt="Screenshot-20251102182946" src="https://github.com/user-attachments/assets/5852ac51-c191-4e99-8726-8cdadc185bfd" />
<img width="415" height="156" alt="Screenshot-20251102183146" src="https://github.com/user-attachments/assets/3f3fc411-2151-4478-9b59-ff74c2b0a724" />
<img width="294" height="104" alt="Screenshot-20251102183228" src="https://github.com/user-attachments/assets/82c51715-c4b4-4e8f-8e79-28e0e5bc24cb" />
<img width="385" height="108" alt="Screenshot-20251102183248" src="https://github.com/user-attachments/assets/37076989-c121-4d31-90c9-0f4704c37e34" />

Axis XY should be used as upper case
<img width="385" height="401" alt="Screenshot-20251102182912" src="https://github.com/user-attachments/assets/aba3623e-5093-496d-a472-75b55282ee30" />

afmenez commented:
- Found many instances of lowercase extensions, will do a separate PR.
- Fixed some of the "..." issues on #11247.  It is complicated to fix the space on "L..." because on the cases appointed, the "..." is not on the original string.
- I will try to install the en-US version to check those "Sentence case" issues later.
- I will try to find all the axis casing issues. I thought I already fixed all of those in a previous PR.

yw4z commented:
Just tried to give you few ideas since you interested to fix text related things
there is a global variable for dots . you can write some of them like "string" + " " + dots. but this will not work on every place

afmenez commented:
I don't believe it is a common practice, I checked some other apps and not a single one add this space... I'll leave it out for now.

Noisyfox commented on localization/i18n/ca/OrcaSlicer_ca.po:
```diff
@@ -4109,7 +4109,7 @@ msgstr ""
 msgstr "Càrrega de filament"
 
-msgid "Motor noise cancellation"
+msgid "Motor noise cancelation"
```
According to Google, I think cancellation is the more widely used one even in American English.

afmenez commented:
Rebased and addressed review comment.

afmenez commented:
Rebased and added more fixes.

Noisyfox reviewed (approved):
LGTM, thanks!
